pkgs/development/tools/simavr/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, avrgcclibc, libelf, which, git, pkgconfig, freeglut, mesa }:
+
+stdenv.mkDerivation rec {
+  name = "simavr-${version}";
+  version = "1.3";
+  enableParallelBuilding = true;
+
+  libPath = stdenv.lib.makeLibraryPath [
+    libelf
+  ];
+
+  src = fetchurl {
+    url = "https://github.com/buserror/simavr/archive/v${version}.tar.gz";
+    sha1 = "b1a875fb23ad819545bcb1889e5fb5470deb09c5";
```

Comment:
Is there a need for sha1 (weak)? If possible please update to sha256. 
Also `fetchFromGitHub` might be the perfect match for this job ;-)